src/main/java/com/selfxdsd/selfweb/api/WalletsApi.java

```diff
@@ -203,4 +207,63 @@ public WalletsApi(final User user, final Self self) {
         }
         return response;
     }
+
+    /**
+     * Activates a Wallet. Returns an array of wallet types which includes the
+     * activated one and the other types deactivated.
+     * @param owner Owner of the project (login of user or org name).
+     * @param name Repo name.
+     * @param type Wallet type (ex: stripe).
+     * @return Array of wallet types which includes the activated one and the
+     * other types deactivated.
+     */
+    @PutMapping(
+        value = "/projects/{owner}/{name}/wallets/{type}/activate",
+        produces = MediaType.APPLICATION_JSON_VALUE
+    )
+    public ResponseEntity<String> activate(
+        @PathVariable final String owner,
+        @PathVariable final String name,
+        @PathVariable final String type){
+
+        final ResponseEntity<String> response;
+        final Project project = this.self.projects().getProjectById(
```

Comment:
@criske Here, we should use the ``user``, in order to access the projects, in order to have only the user's projects.

So ``this.user.projects()...``

Right now, it's actually a security bug, because any authenticated user can access the wallets of any project of any other user.

Reply:
@amihaiemil Should I leave a todo for the other wallet endpoints that are using `self.projects`?

Reply:
Yes please :D